src/main/kotlin/org/kamiblue/client/gui/hudgui/elements/misc/CPS.kt

```diff
@@ -0,0 +1,41 @@
+package org.kamiblue.client.gui.hudgui.elements.misc
+
+import net.minecraftforge.fml.common.gameevent.TickEvent
+import org.kamiblue.client.event.SafeClientEvent
+import org.kamiblue.client.event.events.MouseClickEvent
+import org.kamiblue.client.gui.hudgui.LabelHud
+import org.kamiblue.client.setting.GuiConfig.setting
+import org.kamiblue.client.util.threads.safeListener
+import org.kamiblue.event.listener.asyncListener
+
+object CPS : LabelHud(
+    name = "CPS",
+    category = Category.MISC,
+    description = "Display your clicks per second."
+) {
+
+    var clicks = HashSet<Long>()
+
+    private val measurementTime by setting("Measurement Time (m)", 1000, 500..5000, 100)
```

Comment:
I'd prefer this be called Average Speed Time, like PlayerSpeed does, and you should use the description to indicate the unit of time, such as `Time smoothing, in ms`.